proto/document/v1/document.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+package nitric.document.v1;
+
+import "google/protobuf/struct.proto";
+
+// protoc plugin options for code generation
+option go_package = "nitric/v1;v1";
+option java_package = "io.nitric.proto.document.v1";
+option java_multiple_files = true;
+option java_outer_classname = "Documents";
+option php_namespace = "Nitric\\Proto\\Document\\V1";
+option csharp_namespace = "Nitric.Proto.Document.v1";
+
+// Service for storage and retrieval of simple JSON keyValue
+service DocumentService {
```

Comment:
All the other services don't have the word "Service" in their name. I think it should be consistent with the others and just be Document?

Reply:
We discussed this and we're going to go the other way (add Service prefix to all existing services). It helps reduce class name collisions for generated code